react-sortable-hoc/index.d.ts

```diff
@@ -1,6 +1,6 @@
 // Type definitions for nes 0.0.7
 // Project: https://github.com/clauderic/react-sortable-hoc
-// Definitions by: Ivo Stratev <https://github.com/NoHomey>
+// Definitions by: Andrew de Waal <https://github.com/andrewdewaal>
```

Comment:
Here is an example how it should [look](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3153e0fb9982cd5135d141a0b6d8aa84e33be4e8/jest/index.d.ts#L3)